ChildProject/cmdline.py

```diff
@@ -513,10 +513,13 @@ def compute_durations(args):
         right_on="recording_filename",
     )
     recordings["duration"].fillna(0, inplace=True)
-    recordings["duration"] = recordings["duration"].astype(int)
-    recordings.to_csv(
-        os.path.join(project.path, "metadata/recordings.csv"), index=False
-    )
+    recordings["duration"] = recordings["duration"].astype("Int64")
+    
+    project.recordings = recordings.copy()
+    project.write_recordings()
+#    recordings.to_csv(
+#        os.path.join(project.path, "metadata/recordings.csv"), index=False
+#    )
```

Comment:
is it typically ok to leave old code commented out? or is it better to remove it?

Reply:
I think sometimes it is useful if the old code has a different approach and knowing the original helps.
But here clearly not, this is quite dirty way of writing the file and it is not good to keep it there imo.

Reply:
removed